code/modules/mob/living/basic/space_fauna/bear/bear.dm

```diff
@@ -97,8 +100,8 @@
 	armour_penetration = 20
 	health = 120
 	maxHealth = 120
-	armored = TRUE
 	gold_core_spawnable = HOSTILE_SPAWN
+	armored = TRUE
 
 /obj/item/bear_armor
```

Comment:
would you mind splitting this out into a file in the items folder or something? i really do rather that all of the stuff in this file is just direct mob code